packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -685,4 +685,47 @@ describe("/rows", () => {
       expect(row._id).toEqual(existing._id)
     })
   })
+
+  describe("view search", () => {
+    function priceTable(): Table {
+      return {
+        name: "table",
+        type: "table",
+        schema: {
+          Price: {
+            type: FieldType.NUMBER,
+            name: "Price",
+            constraints: {},
+          },
+          Category: {
+            type: FieldType.STRING,
+            name: "Category",
+            constraints: {
+              type: "string",
+            },
+          },
+        },
+      }
+    }
+
+    it("returns table rows from view", async () => {
+      const table = await config.createTable(priceTable())
+      const rows = []
+      for (let i = 0; i < 10; i++)
```

Comment:
Usually always add {} to any if. Have seen some cases before where someone accidentally adds a second statement inside an unwrapped if that had side effects